airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/class_types_registry.py

```diff
@@ -62,9 +63,10 @@
     "HttpRequester": HttpRequester,
     "InterpolatedBoolean": InterpolatedBoolean,
     "InterpolatedString": InterpolatedString,
-    "JsonSchema": JsonSchema,
```

Comment:
is there a worry that this might break existing users?

Reply:
I scanned through our existing low code implementations and i'm going to migrate them to the new name `JsonFileSchemaLoader` after the CDK is published first. However, on second thought, I think instead of doing the rename now, we'll just bundle this up into a series of cosmetic changes we have slated in November. Even though Hacktober fest is over, some contributors are still making adjustment and we don't want to add unnecessary friction if it can wait